docs/devel/translations.rst

```diff
@@ -17,10 +17,35 @@ This can be done using :guilabel:`Start new translation` in :ref:`component`.
 
 .. note::
 
-    Project admins can always start translation within Weblate directly.
+    Project admins can always start translations within Weblate directly.
 
 Language files added manually to the VCS are added to the component when Weblate updates
-the repository. About repository update settings, see :ref:`update-vcs`.
+the repository. ref:`update-vcs` has info on updating repository settings.
+
+.. _removing-translation:
+
+Removing existing translations
+------------------------------
+
+Components or the projects they are in can be removed in their entirety
+from :guilabel:`Removal` found in the administration tab shown to administrators
+of each project or component. Languages can be removed by navigating to the
+one to delete and finding the menu under Tools > Removal.
```

Comment:
* `the projects` — the is probably excess
* `they are in` is a typo
* :guilabel:`Removal` is not in a "tab" I'd call it menu.

I'd suggest to rephrase it like this:

> Components and projects can be removed in their entirety from their page with :guilabel:`Manage` > :guilabel:`Removal`. Note that the :guilabel:`Manage` menu itself is only shown to the administrators of the project. Languages can be removed in the same manner by :guilabel:`Tools` > :guilabel:`Removal`.
>
>

Reply:
I've changed the suggestion because I was wrong a bit about the `Manage` menu...

Reply:
I reworded this part before seeing this comment, I am sorry. I believe it is alright now.